plugin.xml

```diff
@@ -19,6 +19,11 @@
     <platform name="android">
         <config-file target="res/xml/config.xml" parent="/*">
             <preference name="webView" value="org.crosswalk.engine.XWalkWebViewEngine" />
+            <preference name="lib_mode" value="embedded" />
+        </config-file>
+
+        <config-file target="AndroidManifest.xml" parent="/manifest">
+            <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"></uses-permission>
```

Comment:
I think the default embedded mode doesn't need the `android.permission.WRITE_EXTERNAL_STORAGE` permission.